### ISSUE TITLE
survival script for xposed

### ISSUE THREAD

**Issue by theScrabi** [micro-a5k/microg-unofficial-installer#10]
I'm using microg together with xposed.
However when I install an update of LOS, microG keeps be installed because of the survival script here, but xposed is something I always have to install again.

I wanted to ask if you could put an optional survivalscript in here for people who use xposed.

**Comment by ale5000-git** [comment]
I will do it in the future but as separate flashable zip, I'll keep the issue as reminder.

**Comment by ale5000-git** [comment]
@theScrabi
Now that most have switched to Magisk, is Xposed still in wide use?

Beside that if needed it will be a separate flashable zip.

**Comment by theScrabi** [comment]
No, Xposed itself is discontinued. Now there is a fork based on magisk that tries to carry that on. So no need for a survival script anymore.
